Sprint-3/2-mandatory-rewrite/2-is-proper-fraction.test.js

```diff
@@ -5,7 +5,17 @@ test("should return true for a proper fraction", () => {
 });
 
 // Case 2: Identify Improper Fractions:
+test("should return false for an improper fraction", () => {
+  expect(isProperFraction(7, 3)).toEqual(false);
+});
 
 // Case 3: Identify Negative Fractions:
+test("should return true for a negative fraction", () => {
+  expect(isProperFraction(-2, 3)).toEqual(true);
```

Comment:
Stretch question: this library also has `.toBeTruthy()` - why might we **not** want to use this?

[Doc](https://jestjs.io/docs/expect#tobetruthy)